client/src/components/Header/index.tsx

```diff
@@ -71,9 +88,13 @@ export class Header extends React.Component<
           <UniversalNav
             displayMenu={displayMenu}
             fetchState={fetchState}
+            displayLanguageMenu={displayLanguageMenu}
```

Comment:
nitpick: we might need to change `displayLanguageMenu` to `isLanguageMenuDisplayed` or something so it is not confused with `showLanguageMenu`.